staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 // TODO(knverey): remove this hardcoding once kubectl being built with module support makes BuildInfo available.
-const kustomizeVersion = "v4.5.4"
+const kustomizeVersion = "v4.5.7"
```

Comment:
non-blocking for this PR, but our build is now building in module mode, so I *think* this TODO can be resolved (in a separate PR)

Reply:
FYI @KnVerey